test/quantization/core/test_quantized_op.py

```diff
@@ -3661,6 +3661,93 @@ def test_embedding_bag_2d_indices(self):
         torch.testing.assert_close(result, qresult, atol=0.05, rtol=1e-3)
 
 
+class TestDynamicQuantizedConv(TestCase):
```

Comment:
nit can we put this to TestQuantizedOps or TestDynamicQuantizedOps? we'd like to reduce the TestCases in this file actually

Reply:
ok, will do

Reply:
actually I'm going to leave as is, makes more sense to factor that out in a different PR